docs/excel/custom-functions-overview.md

```diff
@@ -16,9 +16,10 @@ function add42 (a, b) {
 
 Custom functions are now available in preview. Follow these steps to try them:
 
-1.  Install Office 2016 for Windows and join the [Office Insider](https://products.office.com/en-us/office-insider) program.
+1.  Join the [Office Insider](https://products.office.com/en-us/office-insider) program to install the version of Excel 2016 that's required for custom functions on your computer.
```

Comment:
@saunders77 -- does it make sense to mention a specific build number here?  i.e., something like:  

1. Join the [Office Insider](https://products.office.com/en-us/office-insider) program to install the version of Excel 2016 that's required to run custom functions on your computer. (Office build number 8711 or later is required.)